benchexec/tools/graves-par.py

```diff
@@ -0,0 +1,78 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.coveriteam as coveriteam
+import benchexec.result as result
+from benchexec.tools.template import UnsupportedFeatureException
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import re
+
+class Tool(coveriteam.Tool):
+    """
+    Tool infor for Graves, a verifier selector based on Graph Neural Networks 
+    We inherit Coveriteam's infrastructure. 
+    """
+
+    REQUIRED_PATHS = [
+        "coveriteam",
+        "bin",
+        "lib",
+        "depends",
+        "predict",
+        "predict/build",
+    ]
+
+    def name(self):
+        return "Graves-Par"
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("graves.sh")
+
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
+        )
+
+    def cmdline(self, executable, options, task, rlimits):
+        """
+        Modified from the CoVeriTeam definition
+        """
+
+        if task.property_file:
+            options += [task.property_file]
+        else:
+            raise UnsupportedFeatureException(
+                "Can't execute Graves: "
+                "Specification is missing."
+            )
+
+        options += [task.single_input_file]
+
+        return [executable] + options
```

Comment:
As I expect that you want to submit Graves-Par to SV-COMP: Are you sure that you do not need the data model?

Reply:
Correct. We initially tested on 32bit only